### ISSUE TITLE
Problem with new data event and search

### ISSUE THREAD

**Issue by phls** [pyvideo/pyvideo#91]
Hi,

Pyvideo is really nice and I'm it to a new project in Brazil.
I installed and it's running.
http://videos.softwarelivre.org
(the videos there now are to tests)

But I have two issues:

1)
When I create a new event in pyvideo/data and I run "make html", I have this:
CRITICAL: ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S %z'
Makefile:57: recipe for target 'html' failed
make: *** [html] Error 1

If I run "make html" with the events in github.com/pyvideo/data everything is ok. The problem is to create a new one.

2)
The search box is not working. 
Do I miss install something? Maybe something with jquery.

I'm using ubuntu server.

Thanks!

**Comment by phls** [comment]
I found the issue 1. I had to fork python/data, clone it, create a new event there, git push to my fork, and git pull to my pyvideo. 
The "make html" looking for events on github.

**Comment by zerok** [comment]
@phls Sorry for the late response. The search box won't work for your site simply because it is a custom service tailored to pyvideo.org. If you're curious you can find the source code here: https://github.com/zerok/pyvideosearch

**Comment by phls** [comment]
@zerok  

Can I have pyvideosearch in my server to use in my site?
I installed but it's not working. 
http://videos.softwarelivre.org

Is pyvideo in one server, and pyvideosearch in other?

**Comment by zerok** [comment]
@phls Yes, pyvideosearch is hosted on a different server. If you want to use the service on your site, you will have to update the list of allowed hosts. Otherwise browsers will reject XHR calls there.

**Comment by phls** [comment]
@zerok can we talk by email about some details?

**Comment by jonafato** [comment]
We're in the process of closing some resolved issues. The date format error was recently resolved by #370, and it sounds like the search questions may have been discussed out of band. I'm closing this issue as resolved, but if there are other search issues, we can reopen the discussion.
